packages/cli/test/integration-1.test.ts

```diff
@@ -834,9 +838,9 @@ test.skip('deploy `api-env` fixture and test `vercel env` command', async () =>
     const homeUrl = localhost[0];
     const homeRes = await fetch(homeUrl);
     const homeJson = await homeRes.json();
-    expect(homeJson['VERCEL']).toBeUndefined();
+    expect(homeJson['VERCEL']).toBe('1');
     expect(homeJson['VERCEL_URL']).toBe(localhostNoProtocol);
-    expect(homeJson['VERCEL_ENV']).toBeUndefined();
+    expect(homeJson['VERCEL_ENV']).toBe('development');
```

Comment:
These have values in dev now because of https://github.com/vercel/vercel/pull/12358